assembl/static/js/app/views/ideaPanel.js

```diff
@@ -125,6 +155,40 @@ var IdeaPanel = AssemblPanel.extend({
     'click .js_openTargetInPopOver': 'openTargetInPopOver'
   },
 
+  _calculateContentHeight: function(domObject, imageDomObject){
+    var contentPanelPosition = $(window).height() / 3;
+    var imgHeight = imageDomObject.height();
+    if(imgHeight > contentPanelPosition){
+      domObject.css('top', contentPanelPosition);
+    }
+    else{
+      domObject.css('top', imgHeight);
+    }
+  },
+  /*
+    Manages the spacing at the top of the ideaPanel, depending on the panel having an
+    attachment or not.
+   */
+  checkContentHeight: function(){
```

Comment:
Have you tried obtaining the same results in terms of image positioning/sizing/cropping using CSS only (no javascript)? I feel it would be much more robust and clean than using javascript.